webservices/args.py

```diff
@@ -456,8 +456,8 @@ def _deserialize(self, value, attr, data):
 
 schedule_c = {
     'committee_id': fields.List(IStr, description=docs.COMMITTEE_ID),
-    #'candidate_name': fields.List(fields.Str, description='Name of candidate'),
-    #'loaner_name': fields.List(fields.Str, description='Name of loaner'),
+    'candidate_name': fields.List(fields.Str, description='Name of candidate'),
+    'loaner_name': fields.List(fields.Str, description='Name of loaner'),
```

Comment:
Should we use `docs.CANDIDATE_NAME` for the description of `candidate_name` and create a `docs.LOANER_NAME` entry for the `loaner_name` description?